Infrastructure.Data/Mappings/CustomerEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+﻿using DomainModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Data.Mappings
+{
+    public class CustomerEntityTypeConfiguration : IEntityTypeConfiguration<Customer>
+    {
+        public void Configure(EntityTypeBuilder<Customer> builder)
+        {
+            builder.ToTable("Customer");
```

Comment:
Tabelas sempre no plural porque é uma lista de Customer
```suggestion
            builder.ToTable("Customers");
```